### PR TITLE
[PORT]: Elementizes and Greyscales blood decals/overlays

### PR DESCRIPTION
**Pull request by cowbot92**
Ports this: https://github.com/shiptest-ss13/Shiptest/pull/1882 from shiptest (Which ports a bunch of other stuff from bee/tg) 

# Document the changes in your pull request

Changes how blood works, now you can set the color of a species' blood without having to make 90 new sprites
Also it apparently makes the server run faster somehow but Ill never trust people who say that

# Why is this good for the game?

I gave lizards blue blood :)

# Testing

image

# Spriting

refer to testing 

# Wiki Documentation

no

# Changelog

:cl:  

tweak: tweaks how blood works
imageadd: adds new blood
imagedel: removes the old bad blood
experimental: fear the old blood

/:cl:

### CODE REVIEW

**Comment by MommaDragon** [comment]
Okay, now make IPCs have blue coolant for blood that causes toxin damage when you Bloodsucker/Vampire one.

File: code/modules/mob/living/carbon/human/species_types/IPC.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	id = "ipc"
 	say_mod = "states" //inherited from a user's real species
 	sexes = FALSE
-	species_traits = list(NOTRANSSTING,NOEYESPRITES,NO_DNA_COPY,NOZOMBIE,MUTCOLORS,NOHUSK,AGENDER,NOBLOOD,NO_UNDERWEAR)
```
Comment (Moltijoe) [inline code/modules/mob/living/carbon/human/species_types/IPC.dm:8]: Removing NOBLOOD changes a number of things

- prevents rolling bloodsucker and vampire
- allows bloodloss to kill them
- gives them a heart and makes them require a heart to live
- blood cult

It changes far too many things about the species
if you want a robot species to have weird blood, use preternis

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/ethereal.dm [hunk 2]
```diff
@@ -8,7 +8,7 @@
 	mutantlungs = /obj/item/organ/lungs/ethereal
 	mutantstomach = /obj/item/organ/stomach/cell/ethereal
 	mutantheart = /obj/item/organ/heart/ethereal
-	exotic_blood = /datum/reagent/consumable/liquidelectricity //Liquid Electricity. fuck you think of something better gamer
+	exotic_bloodtype = "E"
```
Comment (Moltijoe) [inline code/modules/mob/living/carbon/human/species_types/ethereal.dm]: at no point do you actually define an "E" /datum/blood_type

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/IPC.dm (hunk 1 above)
Comment (cowbot92) [reply code/modules/mob/living/carbon/human/species_types/IPC.dm:8]: fixed it

File: code/modules/mob/living/carbon/human/species_types/ethereal.dm (hunk 2 above)
Comment (cowbot92) [reply code/modules/mob/living/carbon/human/species_types/ethereal.dm]: resolved

**Comment by cowbot92** [comment]
I dont want to deal with these weird runtimes, someone else can do this if they want

**Pull request closed without merging**